Makefile

```diff
@@ -25,7 +33,29 @@ ifdef USE_GIT
   endif
 endif
 
-CXXFLAGS:=$(CXXFLAGS) -I$(SNAPPY_SRC_DIR)
+ifdef ENABLE_BITSHUFFLE
```

Comment:
This flag will be unnecessary if we can use the universal code of bitshuffle. Is that correct?